node.gyp

```diff
@@ -187,14 +187,15 @@
       'lib/internal/priority_queue.js',
       'lib/internal/process/esm_loader.js',
       'lib/internal/process/execution.js',
+      'lib/internal/process/parse_args.js',
       'lib/internal/process/per_thread.js',
       'lib/internal/process/policy.js',
       'lib/internal/process/promises.js',
-      'lib/internal/process/warning.js',
-      'lib/internal/process/worker_thread_only.js',
       'lib/internal/process/report.js',
       'lib/internal/process/signal.js',
       'lib/internal/process/task_queues.js',
+      'lib/internal/process/warning.js',
+      'lib/internal/process/worker_thread_only.js',
       'lib/internal/querystring.js',
```

Comment:
As I mentioned in another comment, please keep the order here so it doesn't affect the git history.

Reply:
this is done